src/Prism.Plugin.Popups/Dialogs/KnownPopupDialogParameters.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Prism.Plugin.Popups.Dialogs
+{
+    public static class KnownPopupDialogParameters
+    {
+        /// <summary>
+        /// Controls whether the popup dialog action (show/close) shall be animated
+        /// </summary>
+        public const string Animated = "animated";
+
+        /// <summary>
+        /// Controls whether the popup page has show and close animations enabled
+        /// </summary>
+        public const string IsAnimationEnabled = "isAnimationEnabled";
+    }
```

Comment:
I would say this is really quite confusing to have two different properties to control animation.

Reply:
I agree that it can be confusing, but these also provide two different ways of controlling animation. We could remove `KnownPopupDialogParameters.IsAnimationEnabled` to make it easier, but would also sacrifice some of the flexibility. That way, controlling the overall popup animations would only be possible by addidng the BindableProperty `PopupDialogLayout.IsAnimationEnabled` to the XAML/Codebehind of the Dialog.